com.unity.ml-agents/Runtime/Actuators/ActionSpaceDef.cs

```diff
@@ -0,0 +1,71 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using Unity.MLAgents.Policies;
+
+namespace Unity.MLAgents.Actuators
+{
+    public readonly struct ActionSpaceDef
+    {
+
+        public readonly SpaceType SpaceType;
+
+        /// <summary>
+        /// An array of branch sizes for our action space.
+        ///
+        /// For an IActuator that uses a Discrete <see cref="SpaceType"/>, the number of
+        /// branches is the Length of the Array and each index contains the branch size.
+        /// The cumulative sum of the total number of discrete actions can be retrieved
+        /// by the <see cref="sumOfDiscreteBrancheSizes"/> property.
+        ///
+        /// For an IActuator with a Continuous <see cref="SpaceType"/>, the number of continuous
+        /// actions is at the 0th index of array.
```

Comment:
Looks like this comment is incorrect since MakeContinuous() passes a null branches arg.